denonavr/denonavr.py

```diff
@@ -307,6 +318,9 @@ def __init__(self, host, name=None, show_all_inputs=False, timeout=2.0,
 
         # Determine receiver type and input functions
         self._update_input_func_list()
+        
+        # Determine device info
+        self.get_device_info()
```

Comment:
What happens if the device is offline at this point in time? Maybe we could check if there are device infos on update.

Reply:
For me personally the receiver is always reachable even if it is off, but I did not think of other models that might not respond if they are in standby.

Will add a check in the update

Reply:
As it is now, if the receiver will not respond to the request you will get a ConnectionError, this is picked up by HomeAssistant and a ConfigEntryNotReady will be thrown. I think that means HomeAssistant will mark the entity as unavailable and will keep retrying untill it gets a connection and then setup the entry.

Reply:
So I don't think this is a problem.

The thing is that the way in which the Config Flow is made right now requires data from the receiver at startup like the mac adress and manufacturer etc. to setup the device entry.

I could store all that data in the config_registry at the moment the Config Flow is setup and reuse that data at a restart of HomeAssistant, but that will require some more coding and time.

I think almost all Denon and Marantz receivers always respond to requests even if they are in standby, so then this is no issue. Otherwise I am pretty sure the component will still be setup as soon as the receiver is turned on for the first time since a HomeAssistant restart.

Reply:
Not everyone is keeping their receivers in standby all the time. Thus I would not assume that the device is running everytime when home-assistant is starting.
There are other places in the code where this seems to be not working all the time. I'll merge und fix this all together